### DISCUSSION TITLE
typewriter mode & focus mode

### DISCUSSION THREAD

**Discussion by care8** [notaapp/talk#414] [Feature Requests]
Hi –

thanks for creating such a nice app :)
one feature I'd love is the typewriter mode available in vscode
and the focus mode originally available in caret.
both are available within typora and would be really nice to be able to use in nota as well

Thank you!

![typewriter](https://github.com/notaapp/talk/assets/12786422/37a527e9-b8be-432a-8f8a-9e4802eca1bf)
![focus](https://github.com/notaapp/talk/assets/12786422/c3f13693-cbac-445f-86ee-656fda466aab)

**Comment by astoilkov** [comment]
We're already experimenting with typewriter mode. We have an internal experiment. We'll probably ship it in the next 2 versions. We're still wondering about focus mode. Let's see if people will vote for focus mode here.

**Reply by greenieves** [reply]
I would deeply appreciate a focus mode
